### ISSUE TITLE
Only one sample or multiple samples can be used in wtdbg2?

### ISSUE THREAD

**Issue by Neo-xbx-00** [ruanjue/wtdbg2#256]
Hi, following are my two question about the use of wtdbg2.
I wonder whether the multiple ccs sequencing files can be derived from several samples, or are supposed to be derived from only one individual?
As the animal we would like to get genome is tiny in size, only one animal probably can't produce enough sequencing data. We hope to know if we can use several animals for sequencing and utilize the data from multile sources for assembly?
By the way, this kind of animal is wild rather than domestic. The results of genome survey demonstrate the heterozygosity is high (~0.5%); whether this can cause further influence to genome assembly?

**Comment by ruanjue** [comment]
Yes, wtdbg2 can process high heterozygous genome. But the continguity will be lower than pure individual. Also, try to build low input library for PacBio sequencing.
